session/SessionHandler.php

```diff
@@ -19,7 +19,7 @@ interface SessionHandlerInterface
      * </p>
      * @since 5.4
      */
-    public function close();
+    public function close(): bool;
```

Comment:
Corresponding methods from `\SessionHandler` should also be updated:
https://github.com/JetBrains/phpstorm-stubs/blob/master/session/SessionHandler.php#L171
https://github.com/JetBrains/phpstorm-stubs/blob/master/session/SessionHandler.php#L179
https://github.com/JetBrains/phpstorm-stubs/blob/master/session/SessionHandler.php#L191
https://github.com/JetBrains/phpstorm-stubs/blob/master/session/SessionHandler.php#L206
https://github.com/JetBrains/phpstorm-stubs/blob/master/session/SessionHandler.php#L219
https://github.com/JetBrains/phpstorm-stubs/blob/master/session/SessionHandler.php#L232
https://github.com/JetBrains/phpstorm-stubs/blob/master/session/SessionHandler.php#L251

https://www.php.net/manual/en/class.sessionhandler.php

Reply:
Oh. Missed them. Updated.